File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.js
function solution(a, b) {
    let str_a = a.toString()
    let str_b = b.toString()

    if (Number(str_a + str_b) > Number(str_b + str_a)) {
        return Number(str_a + str_b)
    } else {
        return Number(str_b + str_a)
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        StringBuilder sb1 = new StringBuilder();
        String str1 = sb1.append(a).append(b).toString();

        StringBuilder sb2 = new StringBuilder();
        String str2 = sb2.append(b).append(a).toString();
        
        int num1 = Integer.parseInt(str1);
        int num2 = Integer.parseInt(str2);
        
        return num1 > num2 ? num1 : num2;
    }
}